administrator/components/com_modules/layouts/joomla/form/field/modulespositionedit.php

```diff
@@ -85,4 +86,20 @@
 		)
 	);
 ?></joomla-field-fancy-select>
-
+<?php 
+	echo HTMLHelper::_(
+		'bootstrap.renderModal',
+		'Modal_position',
+		array(
+			'title'       => 'Select position from template',
+			'url'         => Uri::root() . '?pm=1&edit=1',
+			'height'      => '400px',
+			'width'       => 'auto',
+			'bodyHeight'  => 70,
+			'modalWidth'  => 95,
+			'footer'      => '<button type="button" class="btn btn-secondary" data-bs-dismiss="modal">'
+								. Text::_('JLIB_HTML_BEHAVIOR_CLOSE') . '</button>',
+		)
+	);
+?>
+<a data-bs-toggle="modal" role="button" data-bs-target="#Modal_position">Select position from template</a>
```

Comment:
![image](https://user-images.githubusercontent.com/53610833/121584617-4a238100-ca4f-11eb-99a2-3519cdba0f8a.png)

A preview link displays a modal that allows users to select a position.

The functionality for this is still a Work in Progress